__tests__/app.test.js

```diff
@@ -14,33 +14,44 @@ afterAll(() => {
 });
 
 describe('app', () => {
-  describe('GET /api/topics', () => {
-    test('should return a status :200 ',() => {
+  describe.only('GET /api/topics', () => {
```

Comment:
remove `.only` before committing your code

Reply:
OK will implement all your suggestions. Thanks for your feedback.